### DISCUSSION TITLE
Question about data verification

### DISCUSSION THREAD

**Discussion by HanaO00** [transmission/transmission#7343] [Q&A]
Hello, the tracker tells me that my torrent is 98% complete, I had a power outage, but in the transmission-daemon web interface (V3) it is green, and if I do the verification, transmission tells me 100% complete

Can we trust the verification of local data or if a very small piece of file is missing, transmission will not see it?

**Comment by ile6695** [comment]
I would trust the local verification more than the tracker.

**Reply by HanaO00** [reply]
Thanks for your help @ile6695
